### ISSUE TITLE
Zero gradients for CuDNNLSTM according to Keras TensorBoard callback

### ISSUE THREAD

**Issue by nicolas-ivanov** [keras-team/keras#12125]
Hello! I would like to visualize gradients of a seq2seq model using [Keras Tensorboard callback][1]. If I'm using a regular LSTM cell in my encoder and decoder, I get nice non-zero gradients:

[![enter image description here][2]][2]

However if I change the rnn cell to CuDNNLSTM some gradients turn to zero, which seem to be incorrect:

[![enter image description here][3]][3]

The both models seem to train correctly.

So, what's wrong with visualisations of CuDNNLSTM gradients? Is there a bug in Keras Tensorboard callback?

Code that I am running is a slightly modified Keras lstm_seq2seq example: https://gist.github.com/nicolas-ivanov/1818d6502d5f1496e5fbe14889eddca1

  [1]: https://keras.io/callbacks/#tensorboard
  [2]: https://i.stack.imgur.com/cVcon.png
  [3]: https://i.stack.imgur.com/FasX3.png

---
**Packages**
```
> pip freeze
absl-py==0.7.0
astor==0.7.1
gast==0.2.2
grpcio==1.18.0
h5py==2.9.0
Keras==2.2.4
Keras-Applications==1.0.6
Keras-Preprocessing==1.0.5
Markdown==3.0.1
numpy==1.16.0
protobuf==3.6.1
PyYAML==3.13
scipy==1.2.0
six==1.12.0
tensorboard==1.12.2
tensorflow==1.12.0
tensorflow-gpu==1.12.0
termcolor==1.1.0
Werkzeug==0.14.1
```

---
**Checklist**

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
`pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps`

- [x] Check that your version of TensorFlow is up-to-date. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by nicolas-ivanov** [comment]
@gabrieldemarmiesse any updates on this issue?
